src/gateway/upgrade.md

```diff
@@ -239,7 +239,12 @@ write all other characters without percent-encoding.
 ### Declarative and DB-less
 
 The version number (`_format_version`) of declarative configuration has been bumped to `3.0` for changes on `route.path`.
-Declarative configurations with older versions will be upgraded to `3.0` automatically.
+Declarative configurations with older versions will be upgraded to `3.0` during migrations.
+
+{:.important}
+> Do not `deck sync` 2.8 or earlier declarative configuration files to 3.0.
```

Comment:
```suggestion
> **Do not use `deck sync` to upgrade declarative configuration files from 2.8 or earlier to 3.0.**
```

Reply:
Does not necessarily imply upgrade. Basically telling them not to use their old files with 3.0.